src/single_cell/Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN python -m build --outdir $OUT_DIR --wheel ./src/$MODULE_DIR
 FROM python:3.11-slim
 
 ARG OUT_DIR=/tmp/ugbio
+ARG USERNAME=app
```

Comment:
add comments for the reason to create this user, here and also in the other docker file

Reply:
also we need to add this to readme